helm-xref.el

```diff
@@ -34,12 +37,12 @@
   :prefix "helm-xref-" :group 'helm)
 
 (defface helm-xref-file-name
-  '((t (:foreground "cyan")))
+  '((t (:inherit 'font-lock-builtin-face)))
   "Face for xref file name"
   :group 'helm-xref)
 
 (defface helm-xref-line-number
-  '((t (:inherit 'compilation-line-number)))
+  '((t (:inherit 'helm-buffer-size)))
```

Comment:
Why `helm-buffer-size´ ? Maybe `line-number` ?

Reply:
It is more consistent to see helm-buffer-size colors in *helm buffers*. Users will be more tuned to it, since helm-xref is after all a helm buffer.

Reply:
If you don't agree with my rationale, please feel free to discard my suggestions and check-in what you feel is appropriate. I'm more interested in a consistent outcome in a color-theme rather than a specific value.

Reply:
`helm-occur` uses `helm-grep-lineno`. What do you think ?

Reply:
sure, go for it! But pls check it in. I'm using this and I've manually changed it in my local repo and I'd like to avoid that.